app/routes/api/session.js

```diff
@@ -2,11 +2,40 @@ var express = require('express');
 var passport = require('passport');
 var User = require('../../models/user');
 var router = express.Router();
+var nodemailer = require('nodemailer');
+
+var transporter = nodemailer.createTransport({
+  service: 'Gmail',
+  auth: {
+    user: 'averagejoesmembers@gmail.com',
+    pass: 'mysupersecretpassword'
+  }
+});
 
 // Try to log in with credentials provided in the request.
 // Creates a new session for the user if successful
 router.post('/login', passport.authenticate('local'), function(req, res){
-  res.json({currentUser: req.user});
+  if(req.user.confirmed == "false"){
```

Comment:
I don't know what other way, I have thought about it a lot. Do you have other suggestions? It was the simplest method I could think of.